webapp/portlet/src/main/webapp/idm-users-management/components/UsersManagementList.vue

```diff
@@ -204,6 +213,12 @@ export default {
         align: 'center',
         width: '10%',
         sortable: false,
+      }, {
```

Comment:
is there a way to reduce duplicated code ? 
Can function superHeaders return something like 
headers() +delete column

So that, all the part present in both is not duplicated